round4/team-02.md

```diff
@@ -0,0 +1,124 @@
+# Chrome Extension Idea: Video Insight
+
+## Authors
+
+John Guirguis, Jake Brooks, Aditya Maddipatla, Ashwin Balaji, & Zain Khan
+
+## Problem Statement
+
+Many students struggle with understanding the corpus of research we’re provided to read through due to the building nature of STEM fields. As such, it’s impossible to succeed in understanding later areas of work without first grasping prior research (often this research is the product of concepts taught in fragments throughout multiple different courses in undergrad). One of the key pain points we aim to address with this product would be the need to look into and search up terms or concepts that you encounter as you read new material if you haven’t seen it before. Another would be struggling with dense wordy explanations of ideas that could be tailored for a more visual audience. Many of our users would likely be international students with multi-lingual backgrounds, and it might help them to have concepts re-explained in a language they’re more familiar with. Finally, we would hope to address recurring or important explanations (potentially for harder or more useful concepts) with bookmark abilities 
+
+## Target Audience 
+
+- Students who would like further elaboration or explanation of concepts that are either new or challenging for them, and which would benefit from added context
+
+  - Visual learners who have trouble grasping concepts and processes described in literature
+
+  - Students who are researching new topics or entering a field they’re not familiar with
+
+  - Students who have multi-lingual backgrounds and want to be more familiar with a text by having it explained in a different language than it’s found in
+
+- Educators who are looking for ways to make their material more engaging or visual for students
+
+## Description
+
+Introducing our Chrome plugin, Video Insights! Have you ever come across a word, sentence, or paragraph while browsing the web and wished you had a quick way to dive deeper into its meaning or context? Video Insights is here to enhance your online learning and exploration experience by seamlessly integrating with YouTube to provide you with a curated list of educational videos related to the content you highlight.
+
+With Video Insights, the process is simple yet powerful. Highlight any text on a webpage, right-click, and select the "Search Videos" option from the context menu. Our plugin instantly analyzes the selected text and uses a sophisticated sorting algorithm to search YouTube for the most relevant and informative videos available.
```

Comment:
I like this idea - will you guys be giving a range of videos to choose from, have user preferences of how short or long the video should be or will it be one video when looking it up?

Reply:
Yes we will be a set of videos that are returned not just one. In addition there will be a combination of 3 search paramters that are customizable. These include video language, video length, and ranking criteria.